packages/apify/src/actor.ts

```diff
@@ -512,15 +514,21 @@ export class Actor<Data extends Dictionary = Dictionary> {
      * For more information, see the [Actor Runs](https://docs.apify.com/api/v2#/reference/actor-runs/) API endpoints.
      * @ignore
      */
-    async setStatusMessage(statusMessage: string): Promise<ClientActorRun> {
+    async setStatusMessage(statusMessage: string, options?: { isStatusMessageTerminal?: boolean }): Promise<any> {
```

Comment:
we should revert this bit, the return type was correct here
```suggestion
    async setStatusMessage(statusMessage: string, options?: { isStatusMessageTerminal?: boolean }): Promise<ClientActorRun> {
```